### DISCUSSION TITLE
doris2.0.1 be启动报错

### DISCUSSION THREAD

**Discussion by Marsrick** [apache/doris#24640] [A - General / Q&A]
./start_be.sh: 行 325: 11605 段错误               ${LIMIT:+${LIMIT}} "${DORIS_HOME}/lib/doris_be" "$@" 2>&1 < /dev/null

![image](https://github.com/apache/doris/assets/80306107/76a98c8f-1095-4ce8-b895-afa5b7e06520)

**Comment by Marsrick** [comment]
句柄都已经配置过了。

**Reply by yagagagaga** [reply]
1. Please ensure to download the correct binary package from the official website.
2. Please ensure that your disk has sufficient space.
3. If it still doesn't work, please check the logs in `${DORIS_HOME}/log/be.out`

**Comment by echobigdata** [comment]
您好，请问这个问题解决了吗

**Comment by xiabingbao2023** [comment]
遇到过类似的问题，后来将虚拟机的CPU类型改成hostcpu就可以了
